@navikt/core/react/src/collapsible/parts/Collapsible.Trigger.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef } from "react";
+import { Slot } from "../../util/Slot";
+import { composeEventHandlers } from "../../util/composeEventHandlers";
+import { useCollapsibleContext } from "../Collapsible.context";
+
+export interface CollapsibleTriggerProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  /**
+   * When true, will render element as its child. This merges classes, styles and event handlers.
+   * @default false
+   */
+  asChild?: boolean;
+}
+
+export const CollapsibleTrigger = forwardRef<
+  HTMLButtonElement,
+  CollapsibleTriggerProps
+>(({ children, asChild, onClick, ...props }, ref) => {
```

Comment:
Vi pleier å skrive `...rest`
```suggestion
>(({ children, asChild, onClick, ...rest }, ref) => {
```